content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -0,0 +1,231 @@
+:guilabel:`File` or :guilabel:`URL` from the :guilabel:`Type` drop-down field menu.
+
+.. image:: pdf_quote_builder/document-form-uploaded-pdf.png
+   :align: center
+   :alt: A standard document form with an uploaded pdf in Odoo Sales.
+
+.. note::
+    If a PDF is uploaded, the :guilabel:`Type` field is auto-populated to :guilabel:`File`, and it
+    cannot be modified.
+
+Then, in the :guilabel:`Sales` section, in the :guilabel:`Visible at` field, click the drop-down
+menu, and select either: :guilabel:`Quotation`, :guilabel:`Confirmed order`, or :guilabel:`Inside
+quote`.
+
+- :guilabel:`Quotation`: the document is sent to (and accessible by) customers at any time.
+
+- :guilabel:`Confirmed order`: the document is sent to customers upon the confirmation of an order.
+  This is best for user manuals and other supplemental documents.
+
+- :guilabel:`Inside quote`: the document is included in the PDF of the quotation, between the header
+  pages and the quote table.
+
+.. example::
+   When the :guilabel:`Inside quote` option for the :guilabel:`Visible at` field is chosen, and the
+   custom PDF file, `Sample Builder.pdf` is uploaded, the PDF is visible on the quotation the in the
+   *customer portal* under the :guilabel:`Documents` field.
+
+    .. image:: pdf_quote_builder/pdf-inside-quote-sample.png
+       :align: center
+       :alt: Sample of an uploaded pdf with the inside quote option chosen in Odoo Sales.
+
+Lastly, in the :guilabel:`E-Commerce` section, decide whether or not to :guilabel:`Show on product
+page` on the front-end (in the online store).
+
+.. example::
+   When the :guilabel:`Show on product page` option is enabled, a link to the uploaded document,
+   `Sample Builder.pdf`, appears on the product's page, located on the frontend in the online store.
+
+   It appears beneath a :guilabel:`Documents` heading, with a link showcasing the name of the
+   uploaded document.
+
+    .. image:: pdf_quote_builder/show-product-page.png
+       :align: center
+       :alt: Showing a link to an uploaded document on a product page using Odoo Sales.
+
+Dynamic text values
+-------------------
+
+While creating custom PDFs for quotes, using any standard PDF editor software, certain dynamic texts
+should be utilized throughout the design of the PDF to ensure Odoo can automatically fill in the
+appropriate content accordingly.
+
+Dynamic text values are, essentially, coded placeholders that Odoo fills in automatically, according
+to the specific information related to the quote.
+
+Below are the dynamic text values and what they will represent in the custom PDFs:
+
+- :guilabel:`name`: Sales Order Reference
+- :guilabel:`partner_id_name`: Customer Name
+- :guilabel:`user_id_name`: Salesperson Name
+- :guilabel:`amount_untaxed`: Untaxed Amount
+- :guilabel:`amount_total`: Total Amount
+- :guilabel:`delivery_date`: Delivery Date
+- :guilabel:`validity_date`: Expiration Date
+- :guilabel:`client_order_ref`: Customer Reference
```

Comment:
I would delete this whole section from this part of the doc, and just include the "Product-specific dynamic text values" list in the above "Dynamic text in PDFs" H2 section.

But if you decide to keep this section here, you probably want change the intro paragraph to match the one in the "Dynamic text in PDFs" H2 section, since the intro paragraph here seems to be an older version.

Reply:
I agree, I suggest to change the title of the section and only keep the list of authorized name for form components